core/sector_storage/worker.hpp

```diff
@@ -142,6 +142,8 @@ namespace fc::sector_storage {
 
     virtual outcome::result<std::vector<primitives::StoragePath>>
     getAccessiblePaths() = 0;
+
+    virtual void ping(std::function<void(const bool &resp)> cb) = 0;
```

Comment:
unnecessary reference may be omited
```suggestion
    virtual void ping(std::function<void(bool)> cb) = 0;
```

Reply:
bool is taking 1 byte of memory, why copying == referencing ?

Reply:
If copy and reference are equal, we prefer copy

Reply:
Referencing takes 4 bytes